pkg/fanal/artifact/image/image_test.go

```diff
@@ -488,1275 +470,1261 @@ func TestArtifact_Inspect(t *testing.T) {
 			artifactOpt: artifact.Option{
 				LicenseScannerOption: analyzer.LicenseScannerOption{Full: true},
 			},
-			missingBlobsExpectation: cache.ArtifactCacheMissingBlobsExpectation{
-				Args: cache.ArtifactCacheMissingBlobsArgs{
-					ArtifactID: "sha256:33f9415ed2cd5a9cef5d5144333619745b9ec0f851f0684dd45fa79c6b26a650",
-					BlobIDs: []string{
-						"sha256:4a26915356c961f038d5a7b7f73f24cd1eec53dcf6fdeecd39b310ddc066faec",
-						"sha256:e23e1d428a2a4ca9607cde5c556f744c7e9f3a1d3bfe835707c0fea107caf453",
-						"sha256:b8ae022ed4f8b8bf827c04a825c2e6998217581d44c0b28b59a4e66ca65bbaa5",
-						"sha256:8c51dcc708602d983f3f0507f0d26de609819c4391db92497639417e54378d11",
-					},
-				},
-				Returns: cache.ArtifactCacheMissingBlobsReturns{
-					MissingBlobIDs: []string{
-						"sha256:4a26915356c961f038d5a7b7f73f24cd1eec53dcf6fdeecd39b310ddc066faec",
-						"sha256:e23e1d428a2a4ca9607cde5c556f744c7e9f3a1d3bfe835707c0fea107caf453",
-						"sha256:b8ae022ed4f8b8bf827c04a825c2e6998217581d44c0b28b59a4e66ca65bbaa5",
-						"sha256:8c51dcc708602d983f3f0507f0d26de609819c4391db92497639417e54378d11",
-					},
+			setupCache: func(t *testing.T) cache.Cache {
+				c := cache.NewMemoryCache()
+				c.PutArtifact("sha256:33f9415ed2cd5a9cef5d5144333619745b9ec0f851f0684dd45fa79c6b26a650", types.ArtifactInfo{
```

Comment:
nit:
I know `c.PutArtifact` can't return an error.
But is it possible to wrap this in `require.NoError` like in [applier_test](https://github.com/aquasecurity/trivy/blob/a9ef9c748216c2e5b8cf46400e33848bde3baeb8/pkg/fanal/applier/applier_test.go#L50) to avoid the warning and make test cases look the same.

Reply:
I just forgot to add it. Thanks for catching it.
https://github.com/aquasecurity/trivy/pull/8403/commits/8dd0f2df1fde81a61e1caeef6b5f4ad78adaa927

Reply:
Somewhere else?
https://github.com/aquasecurity/trivy/pull/8403/commits/7f6b7eebb73486ce435a64c9470bd70b701b1791

Reply:
now everything is correct, thank you!